tests/src/drivers/test_driver_key_management.c

```diff
@@ -125,7 +125,7 @@ static size_t mbedtls_test_opaque_get_base_size()
  * The argument wrapped_key_buffer_length is filled with the wrapped
  * key_size on success.
  * */
-static psa_status_t mbedtls_test_opaque_wrap_key(
+psa_status_t mbedtls_test_opaque_wrap_key(
```

Comment:
I am not totally sure why the wrap function is no longer static. It is not used in the tests I don't think. Is this just to keep it consistent with the unwrap function?

Reply:
Oh, you are totally right. I probably moved both functions from `static` to public, but actually only the unwrap function is needed for opaque drivers. I'm reverting the change for the wrapping key.